src/repository/tvShowRepository.js

```diff
@@ -3,14 +3,18 @@ import transformer from '../transformer/tvShowTransformer';
 
 const API_BASE_URL = 'http://api.tvmaze.com';
 
-const tvShowRepository = {
-  search: query => fetchJson(`${API_BASE_URL}/search/shows?q=${query}`).then(transformer.transformCollection),
-  findById: id => fetchJson(`${API_BASE_URL}/shows/${id}`).then(transformer.transformItem),
-  findByIds: (ids) => {
-    const promises = ids.map(id => fetchJson(`${API_BASE_URL}/shows/${id}`).then(transformer.transformItem));
+const findById = id => fetchJson(`${API_BASE_URL}/shows/${id}`).then(transformer.transformShow);
+const findSeasons = id => fetchJson(`${API_BASE_URL}/shows/${id}/seasons`);
+const findEpisodes = id => fetchJson(`${API_BASE_URL}/shows/${id}/episodes`);
 
-    return Promise.all(promises);
-  },
+const tvShowRepository = {
```

Comment:
to już bym chyba albo wszystkie deklarował wewnątrz tego obiektu albo wszystki na zewnątrz bo tak się znacznie gorzej czyta, nie rozumiejąc czemu niektóre są w obiekcie a niektóre nad nim.